tests/image/imageengine.js

```diff
@@ -47,6 +47,16 @@ describe( 'ImageEngine', () => {
 
 				expect( editor.getData() ).to.equal( '<figure class="image"><img src="foo.png"></figure>' );
 			} );
+
+			it( 'should convert with srcset attribute and add sizes attribute', () => {
+				setModelData( document, '<image src="foo.png" alt="alt text" srcset="small 148w, big.png 1024w"></image>' );
```

Comment:
Just to be clear – was it intentional that `small` has no extension?

If it's meant to test something, it'd be better if that was a separate test. But I guess that it's just a typo, in  which case it'd be better to add the extension.